webrender_traits/src/font.rs

```diff
@@ -47,6 +47,15 @@ pub enum FontRenderMode {
     Subpixel,
 }
 
+const FIXED16_SHIFT: u32 = 16;
+
+// This matches the behaviour of SkScalarToFixed
+fn f32_truncate_to_fixed16(x: f32) -> u32
+{
+    let fixed1 = (1 << FIXED16_SHIFT) as f32;
+    (x * fixed1) as u32
```

Comment:
I'm a bit [worried](http://www.embeddeduse.com/2013/08/25/casting-a-negative-float-to-an-unsigned-int/) about this cast